### ISSUE TITLE
Dependency mismatches @ 1.8.0

### ISSUE THREAD

**Issue by franklixuefei** [DevExpress/devextreme-reactive#1464]
- [x] I have searched this repository's [issues](https://github.com/devexpress/devextreme-reactive/issues) and believe that this is not a duplicate.

### I'm using ...

- [x] React Grid
- [ ] React Chart
- [ ] Vue Grid

### Current Behaviour

```txt
npm WARN @devexpress/dx-react-grid@1.8.0 requires a peer of @devexpress/dx-core@1.8.0-beta.1 but none is installed. You must install peer dependencies yourself.
npm WARN @devexpress/dx-react-grid@1.8.0 requires a peer of @devexpress/dx-react-core@1.8.0-beta.1 but none is installed. You must install peer dependencies yourself.
npm WARN @devexpress/dx-react-grid-material-ui@1.8.0 requires a peer of @devexpress/dx-grid-core@1.8.0-beta.1 but none is installed. You must install peer dependencies yourself.
npm WARN @devexpress/dx-react-grid-material-ui@1.8.0 requires a peer of @devexpress/dx-react-core@1.8.0-beta.1 but none is installed. You must install peer dependencies yourself.
npm WARN @devexpress/dx-react-grid-material-ui@1.8.0 requires a peer of @devexpress/dx-react-grid@1.8.0-beta.1 but none is installed. You must install peer dependencies yourself.
npm WARN @devexpress/dx-grid-core@1.8.0 requires a peer of @devexpress/dx-core@1.8.0-beta.1 but none is installed. You must install peer dependencies yourself.
```

### Expected Behaviour

These warnings should go away.

### Steps to Reproduce (for Bugs)

`npm update`

### Environment

- devextreme-reactive: 1.8.0
- react: 16.5.2
- material-ui: 3.1.2

**Comment by SergeyAlexeev** [comment]
It's just a problem with our internal tools for publishing packages. We'll fix it in the next release.

**Comment by smokja** [comment]
@Methuselah96 Is this the same bug but just on a different version? Did you solve this bug by just installing the peer dependencies manually? Or should I wait for the fix?

**Comment by Methuselah96** [comment]
@smokja It's the same problem, I don't know if the same thing caused it. You can solve it by just ignoring the warnings. Everything should be at v1.8.0 and works fine at v1.8.0. In the next release the warnings will go away.

**Comment by SergeyAlexeev** [comment]
Fixed in [v1.9.0](https://github.com/DevExpress/devextreme-reactive/releases/tag/v1.9.0).
